extensions/KHR/EGL_KHR_platform_wayland.txt

```diff
@@ -93,6 +94,21 @@ New Behavior
     until the next buffer swap. The rationale behind this behavior is to keep
     operations result accurate until the next swap.
 
+    Synchronously within the call to eglSwapBuffers the EGL implementation will
```

Comment:
This doesn't really belong here IMHO. I think it should be injected into the existing definition of eglSwapBuffers() using the usual pseudo-patch semantics of GL/EGL/GLX extension specifications. E.g., "Changes to Chapter 3 of the EGL 1.5 Specification (EGL Functions and Errors)

    Add the following text to subsection 3.10.1 titled "Posting to a
    Window" after paragraph 3 of the description of eglSwapBuffers:

    "When the <surface> refers to a Wayland platform surface, <Your new language here>"

The other language in this spec should arguably be similarly inserted relative to existing spec text, but its more general in nature, so it probably isn't as important to clean that up.

Reply:
It's now "Add the following ..." for this part.